app/helpers/application_helper.rb

```diff
@@ -52,13 +52,15 @@ def table_of_contents_separator(options = {})
   end
 
   def manuscript_link(options = {})
-    druid = options[:value]
+    return if options[:value].blank?
+    druid = options[:value][0]
     document = options[:document]
-    ms_number = document['manuscript_number_tesim']
-    return if druid.blank? || ms_number.blank?
-    return druid if document['format_main_ssim'] != ['Page details']
     title = document['title_full_display']
-    title = title.include?(':') ? title.partition(':')[2] : ms_number[0]
-    link_to title, spotlight.exhibit_solr_document_path(current_exhibit, druid[0])
+    link_title = if (document['format_main_ssim'] == ['Page details']) && title.include?(':')
```

Comment:
We have an accessor on `SolrDocument` that can tell us if the item is a canvas/page details object which might be useful here. https://github.com/sul-dlss/exhibits/blob/0779d173ea2d0861827e252aed21d76fc4e62505/app/models/concerns/canvas_concern.rb#L6-L8

Reply:
👍 sweet